templates/clockwork.toml.j2

```diff
@@ -34,4 +34,20 @@ allocations="*"
 allocations={{ cluster.allocations }}
 {% endif %}
 timezone="{{ cluster.timezone }}"
+organization="{{ cluster.organization }}"
+nbr_cpus={{ cluster.nbr_cpus }}
+nbr_gpus={{ cluster.nbr_gpus }}
+official_documentation="{{ cluster.official_documentation }}"
+mila_documentation="{{ cluster.mila_documentation }}"
+display_order={{ cluster.display_order }}
+remote_user="{{ cluster.remote_user }}"
+remote_hostname="{{ cluster.remote_hostname }}"
+{% if cluster.sacct_enabled %}
+sacct_enabled={{ cluster.sacct_enabled }}
+{% else %}
+sacct_enabled=false
+{% endif %}
+sacct_path="{{ cluster.sacct_path }}"
+sacct_ssh_key_filename="{{ cluster.sacct_ssh_key_filename }}"
```

Comment:
There is already an assumption that there is a single keyfile for all the clusters so it could be a separate global config.

It doesn't matter that much, but it would avoid repetition.